packages/beachball/src/changelog/writeChangelog.ts

```diff
@@ -8,37 +8,39 @@ import { renderChangelog } from './renderChangelog';
 import { renderJsonChangelog } from './renderJsonChangelog';
 import { BeachballOptions } from '../types/BeachballOptions';
 import { isPathIncluded } from '../monorepo/utils';
-import { PackageChangelog } from '../types/ChangeLog';
+import { PackageChangelog, ChangelogJson } from '../types/ChangeLog';
 import { mergeChangelogs } from './mergeChangelogs';
 
-export function writeChangelog(
+export async function writeChangelog(
```

Comment:
we also call this in `performBump`. need to make sure we change that chain of callers to async too

Reply:
fixed, thanks!